packages/web-tracing/src/faroTraceExporter.ts

```diff
@@ -10,7 +10,7 @@ export class FaroTraceExporter implements SpanExporter {
   constructor(private config: FaroTraceExporterConfig) {}
 
   export(spans: ReadableSpan[], resultCallback: (result: ExportResult) => void): void {
-    const traceEvent = createExportTraceServiceRequest(spans, { useHex: true, useLongBits: false });
```

Comment:
I've built the web-sdk again to double check. 
This is working as expected.

Can you try to run the cleanup task, remove node_modules, do a yarn install and built again?

Reply:
Ah did you already update with the main branch?
Maybe some otel update are missing?